inference/enterprise/deployments/entities/steps.py

```diff
@@ -225,7 +234,7 @@ class ObjectDetectionModel(RoboflowModel):
     class_agnostic_nms: Union[Optional[bool], str] = Field(default=False)
     class_filter: Union[Optional[List[str]], str] = Field(default=None)
     confidence: Union[Optional[float], str] = Field(default=0.0)
```

Comment:
Should we also change this to `0.4` to match other library defaults?

Reply:
oh, great catch